modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile registry to access devfiles
+
+You can use the devfiles in the *+Add* flow of the *Developer* perspective to create an application. The *+Add* flow provides a complete integration with the https://registry.devfile.io/viewer[*devfile community registry*]. A devfile is a portable YAML file that describes your development environment without needing to configure it from scratch. Using the *Devfile registry*, you can use a pre-configured devfile to create an application.
```

Comment:
devfile community registry does not seem to be a GUI element, please remove the bold from here.